### ISSUE TITLE
Label cleanup!

### ISSUE THREAD

**Issue by ermo** [getsolus/packages#4117]
1. Are our current labels still fit for purpose?
2. Which new labels could we potentially benefit from?

A couple of labels that immediately came to mind as I was trawling through the packages repo PRs were:
- [x] Blocked (when one PR is blocked by another and the issues/PRs mention each other)
- [x] Needs Rebase (for older issues that are otherwise good to go, but where the original submitter should be the one rebasing)
- [x] Performance (for updates that make something faster w/ the numbers to back it up)

Potentially related:
- https://github.com/getsolus/packages/issues/4116

**Comment by malfisya** [comment]
From the quick glance, the most annoying thing is how unorganized the labels are. We can group them in categories and adding it as  prefix to all of them. For example, instead of `Budgie` we can use `Edition: Budgie` and so on.

**Comment by ermo** [comment]
> From the quick glance, the most annoying thing is how unorganized the labels are. We can group them in categories and adding it as prefix to all of them. For example, instead of `Budgie` we can use `Edition: Budgie` and so on.

Fixed.

**Comment by silkeh** [comment]
I agree with @malfisya's that it would be beneficial to split the labels in categories/scopes. Every category should have the same colour (with different brightness if that is needed, eg: priority), but we can bikeshed that later. This was also the original idea of the colors, but they could be made more explicit. Unfortunately, GitHub doesn't have [scoped labels](https://github.com/orgs/community/discussions/16682), which would make this a lot easier.

My proposal would be to have all label names be in the form `{{ Scope }}: {{ Value }}`, with the following:

| Scope | Color | Values |
|---|---|---|
| Blocked | #cc317c | Upstream, Other
| Edition | #d4c5f9 | Budgie, GNOME, Plasma, Xfce, MATE
| Package | TBD | Request, Update Request, Deprecation/Removal, Does not build
| Priority | keep | Wishlist, Low, Normal, High
| State | keep | Approved, Conditionally Approved, Needs Maintainer, Stale
| Topic | keep | Platform Integration, Plumbing, Tooling, Hardware Support, Kernel, Installation
| Type | keep | Bug, Feature, Chore, Duplicate, Security

Additional labels for Hacktoberfest and new contributors. Maybe make these the same color?

- Good First Issue
- hacktoberfest-accepted
- Help Wanted

The following labels should be removed in my opinion:

- 4.6 ISO Testing: this is what a milestone is for.
- Rejected, Invalid: issues can be closed as 'completed' and as 'rejected', that should be used instead.
- VirtualboxVM: reproduction steps should be in the issue description, not the labels.
- Needs rebase: the PR states (changes requested) should be sufficient here.
- NVIDIA: Not really used.
- Performance: Not really used.
- ISO Blocked: A comment is probably clearer for contributors.
- Confirmed: this is what the project state is for (triage -> ready). Same could be said for 'Needs maintainer', but I'd like to keep that for visibility to contributors.

**Comment by TraceyC77** [comment]
I like Silke's list. The only changes I would suggest are:

Under "Blocked" add ISO 

Replace "Topic" with "Plumbing", I think everything else can fit under that. "Topic" isn't meaningful IMO.

Under Package - Instead of "Does hot build" use "Problem" to keep the label for the Packaging Problem template. (It's used for packages that don't build, that have dependencies missing etc)

**Comment by malfisya** [comment]
I have implemented most of @silkeh 's proposals. I have not added/deleted any of the tags available, so people can give more feedback. What to delete, what to add and what color fits best. Please be aware, editing labels sometimes require editing issue forms and also updating related section in help center. Cheers!

**Comment by TraceyC77** [comment]
Label update work had been done, closing this. Further label work can happen in a new issue.
